src/routes/index.tsx

```diff
@@ -1,10 +1,13 @@
 import { VNode } from "preact";
+import DailyAuthsReport from "../daily-auths-report";
+import DailyDropffsReport from "../daily-dropoffs-report";
 import { Router } from "../router";
 import HomeRoute from "./home-route";
-import ReportRoute from "./report-route";
+import createReportRoute from "./report-route";
 
 export const ROUTES = {
-  "/daily-auths-report/": ReportRoute,
+  "/daily-auths-report/": createReportRoute(DailyAuthsReport, "Daily Auths Report"),
+  "/daily-dropoffs-report/": createReportRoute(DailyDropffsReport, "Daily Dropoffs Report"),
```

Comment:
We got rid of the automated static path generation, so we'll need to add this:

https://github.com/18F/identity-reporting/blob/b871fd559e5a410b0df56e22a69eea33c795e593/bin/generate-routes.js#L5

Reply:
oh good catch, thank you! added in 69d1de3141ad9050b893c348981f5bc47d79967b